src/modules/core/components/ActionsList/ActionsListItem.css

```diff
@@ -0,0 +1,134 @@
+/*
+ * @NOTE Unique values
+ */
+@value mainTextSize: 13px;
+@value popoverWidth: 400px;
+@value popoverDistance: 10px;
+
+.main {
+  display: flex;
+  margin-bottom: 1px;
+  padding: 26px 20px;
+  max-height: 89px;
```

Comment:
I would suggest to set height instead of max-height, to always keep those items same height (even if some of them won't have avatar)

Reply:
Sure